### ISSUE TITLE
[TreeList] The selectable feature cannot be disabled

### ISSUE THREAD

**Issue by svetq** [telerik/kendo-angular#2943]
**Describe the bug**
The selectable feature cannot be disabled via the **enabled** property of the [SelectableSettings](https://www.telerik.com/kendo-angular-ui/components/treelist/api/SelectableSettings/)

 
```
    public settings: SelectableSettings = {
        mode: 'row',
        multiple: true,
        drag: true,
        enabled: false
    };
```

**To Reproduce**
Please check the browser console of the following example:
https://stackblitz.com/edit/angular-znj9vx-epwzdf?file=app/app.component.ts

    Error: isSelected must be a function, but received "null".

**Comment by dtopalov** [comment]
There seems to be a discrepancy caused by using both the selectable directive, and setting `enabled` to false. When the selectable directive is not applied, the selection is disabled and there is no error.

**Comment by dtopalov** [comment]
Fixed in the latest [dev version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/) of the @progress/kendo-angular-treelist package.

**Comment by Raisolution** [comment]
Fix is available in `v3.0.3`
